data/human/campaign events.txt

```diff
@@ -2525,6 +2551,7 @@ event "fwc attack kaus borealis"
 
 event "fwc capture kaus borealis"
```

Comment:
The capture of this system is later reverted, so I wouldn't even bother with updating the uninhabited here. Although if these uninhabited systems are to switch hands, we should do it during the "fw northern expansion" event.

This is what the FW gets after both the southern and northern expansions at the moment. I feel like it makes sense to give the remaining three uninhabited systems here to the Free Worlds, as they connect a route from Seginus to Delta Sagittarri.
![image](https://github.com/endless-sky/endless-sky/assets/17688683/2bca0eb3-ba78-4eb6-bebb-3c18e05be79c)

Reply:
Hmm, I thought the FW kept Kaus Borealis in FWC?

I'll add those uninhabitated systems to Southern expansion.

Reply:
Ok, looks like Kaus Borealis is _meant_ to be returned as well as Cebalrai (`fwc navy retakes cebalrai`), but Kaus Borealis stays FW anyway. I suppose I could fix that in this PR.